### PR TITLE
bap backup script

### PR DESCRIPTION
**Pull request by SpudGunMan**
this is quick idea to address a concern a lot have I would assume?

### CODE REVIEW

**Comment by km4ack** [comment]
like this idea! Need to include pat menu (primary is config, others are .config), wsjtx, and js8call. What about restoring?

**Comment by SpudGunMan** [comment]
I think I have all you listed in there so far look at the file I listed it nicely for viewing.
restoring is tricky it would work to just untar back into your home folder for example however there is a lot of milage issues there.. for example you couldn't exactly update apps that might change config files and restore a year ago with no failures. 

but this would grab all the known config files and they could be put back with archiver etc.. I collected git info to a text file as well, this will help with backup and restore notes 

basically step one backup and find any other locations missed, after that we can also backup a dump of all app versions and with that info a visual check of restore could be done .. could be also put into a GUI "restore this app files" for example could be done I suppose 

anyway step one :)

**Comment by SpudGunMan** [comment]
added patmenu2
